books/migrations/0040_auto_20180402_1158.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 2.0.2 on 2018-04-02 16:58
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtaildocs', '0007_merge'),
+        ('books', '0039_auto_20180301_1107'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='book',
+            name='ally_blurb',
+            field=models.TextField(blank=True),
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='ally_header',
+            field=models.CharField(default='Want more?', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='community_resources_blurb',
+            field=models.TextField(blank=True),
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='community_resources_feature_text',
+            field=models.TextField(blank=True),
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='comp_copy_blurb',
+            field=models.TextField(blank=True),
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='cover_color',
+            field=models.CharField(choices=[('Blue', 'Blue'), ('Deep Green', 'Deep Green'), ('Gold', 'Gold'), ('Gray', 'Gray'), ('Green', 'Green'), ('Light Blue', 'Light Blue'), ('Light Gray', 'Light Gray'), ('Medium Blue', 'Medium Blue'), ('Orange', 'Orange'), ('Red', 'Red'), ('Yellow', 'Yellow')], default='Blue', max_length=255),
```

Comment:
Is this a list of label-value pairs? Can the value be lower-case and hyphenated? That matches the values in the pattern library (e.g. `medium-blue`).